packages/manager/src/features/LoadBalancers/LoadBalancerDetail/ServiceTargets/CreateServiceTargetDrawer.tsx

```diff
@@ -0,0 +1,329 @@
+import { Endpoint, ServiceTargetPayload } from '@linode/api-v4';
+import Stack from '@mui/material/Stack';
+import { useFormik } from 'formik';
+import React from 'react';
+
+import { ActionsPanel } from 'src/components/ActionsPanel/ActionsPanel';
+import { Autocomplete } from 'src/components/Autocomplete/Autocomplete';
+import { SelectedIcon } from 'src/components/Autocomplete/Autocomplete.styles';
+import { Box } from 'src/components/Box';
+import { Divider } from 'src/components/Divider';
+import { Drawer } from 'src/components/Drawer';
+import { FormControlLabel } from 'src/components/FormControlLabel';
+import { FormHelperText } from 'src/components/FormHelperText';
+import { FormLabel } from 'src/components/FormLabel';
+import { InputAdornment } from 'src/components/InputAdornment';
+import { Notice } from 'src/components/Notice/Notice';
+import { Radio } from 'src/components/Radio/Radio';
+import { RadioGroup } from 'src/components/RadioGroup';
+import { TextField } from 'src/components/TextField';
+import { Toggle } from 'src/components/Toggle';
+import { TooltipIcon } from 'src/components/TooltipIcon';
+import { Typography } from 'src/components/Typography';
+import { useServiceTargetCreateMutation } from 'src/queries/aglb/serviceTargets';
+import scrollErrorIntoView from 'src/utilities/scrollErrorIntoView';
+
+import { CertificateSelect } from '../Certificates/CertificateSelect';
+import { AddEndpointForm } from './AddEndpointForm';
+import { EndpointTable } from './EndpointTable';
+
+interface Props {
+  loadbalancerId: number;
+  onClose: () => void;
+  open: boolean;
+}
+
+const algorithmOptions = [
+  {
+    description: 'Sequential routing to each instance.',
+    label: 'Round Robin',
+    value: 'round_robin',
+  },
+  {
+    description:
+      'Sends requests to the target with the least amount of current connections.',
+    label: 'Least Request',
+    value: 'least_request',
+  },
+  {
+    description: 'Reads the request hash value and routes accordingly.',
+    label: 'Ring Hash',
+    value: 'ring_hash',
+  },
+  {
+    description: 'Request are distributed randomly.',
```

Comment:
```suggestion
    description: 'Requests are distributed randomly.',
```
Caught a typo here:
![Screenshot 2023-09-20 at 10 32 01 AM](https://github.com/linode/manager/assets/114685994/20f09e22-6954-4a9a-b12b-fe80eb08eb81)